nextflow.config

```diff
@@ -144,6 +144,7 @@ profiles {
     docker {
         docker.enabled         = true
         docker.userEmulation   = true
+        if (params.use_gpu) { docker.runOptions = '--gpus all' }
```

Comment:
Can you add something similar in the arm profile?

Reply:
Done! @maxulysse I think I have incorporated all the changes you proposed